### PR TITLE
tweak doc and clean pom.xml

### PR DESCRIPTION
**Pull request by JerryShea**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -560,51 +560,6 @@
         </profile>
     </profiles>
 
-    <repositories>
```
Comment (JerryShea) [inline pom.xml:563]: inherited from parent pom

**Review by chroniclekevinpowe — APPROVED** [review]
Love it!

File: docs/CM_Download.adoc [hunk 2]
```diff
@@ -38,15 +60,15 @@ Click here to get the http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22net.open
 
 == Maven Snapshot Download
 If you want to try out the latest pre-release code, you can download the snapshot artifact manually
-from https://oss.sonatype.org/content/repositories/snapshots/net/openhft/chronicle-map/.
+from  https://s01.oss.sonatype.org/content/repositories/snapshots/net/openhft/chronicle-map/.
```
Comment (chroniclekevinpowe) [inline docs/CM_Download.adoc]: Minor feedback: extra space inserted here before https?

Comment (JerryShea) [reply docs/CM_Download.adoc]: thanks, fixed

**Pull request merged**